public/javascripts/Gallery/src/validation/ValidationMenu.js

```diff
@@ -107,9 +140,10 @@ function ValidationMenu(refCard, gsvImage, cardProperties, modal, onExpandedView
      * @param thumbsClick {Boolean} Whether the validation came from clicking the thumb icons.
      * @returns {(function(*): void)|*}
      */
-    function validateOnClick(newValKey, thumbsClick) {
-        return function(e) {
-            if (currSelected !== newValKey) {
+    function validateOnClickOrKeyPress(newValKey, thumbsClick) {
```

Comment:
Can you also make sure that we log to `sg.tracker` differently for clicks vs shortcuts?

Reply:
![image](https://github.com/user-attachments/assets/afff4c7b-655f-4b3a-a108-d7b5d9865306)
![image](https://github.com/user-attachments/assets/16976455-51f0-401d-80f7-870d1e591d9e)

Would these be the areas I need to change? Do I need to do anything else other than just replace the "Click" with "KeyDown"?

Reply:
I think the only other place is when moving to the next label with arrow keys, but it isn't moving to an entirely new page. I'm assuming that we are logging that... If we aren't, then we should, and now would be a good time to add that!

Only other caveat is that I think we typically use "KeyboardShortcut" rather than "KeyDown" in the action names!

Reply:
Got it! Sorry, I confused the NextPage/PrevPage as being the next/prev label for some reason.

I'll make sure to use `KeyboardShortcut_`

I could be completely wrong here, but I can't seem to find any logging for the next/previous labels. I'll create the proper logging for those (both the click version and the KeyboardShortcut version).

Do I need to update some documentation anywhere after adding this logging? I know there's [this](https://github.com/ProjectSidewalk/SidewalkWebpage/wiki/Descriptions-of-Logged-Events).

Reply:
>I could be completely wrong here, but I can't seem to find any logging for the next/previous labels. I'll create the proper logging for those (both the click version and the KeyboardShortcut version).

That wouldn't surprise me! The logging in Gallery is lacking compared to Explore/Validate.

>Do I need to update some documentation anywhere after adding this logging? I know there's [this](https://github.com/ProjectSidewalk/SidewalkWebpage/wiki/Descriptions-of-Logged-Events).

That would be the place to update it, yes! Would need to add a new Gallery column. That page tends to get a bit out of date, but I would encourage every effort to keep it updated!!